src/dto/student-to-import.dto.ts

```diff
@@ -0,0 +1,9 @@
+export class StudentToImportDto {
+  id?: string;
+  email: string;
+  courseCompletion: number;
+  courseEngagment: number;
+  projectDegree: number;
+  teamProjectDegree: number;
+  bonusProjectUrls: string[];
```

Comment:
bonusProjectUrls --> if its dto it can't be string[] 
There is no such a thing in the DB. It can be a string.

Reply:
Sure! I forgot about it.